.devops/deploy-pipelines.yml

```
@@ -0,0 +1,123 @@
+# Azure DevOps pipeline to release a new version and deploy to production.
+
+variables:
+  NODE_VERSION: '12.19.1'
+  YARN_CACHE_FOLDER: $(Pipeline.Workspace)/.yarn
+
+parameters:
+  - name: 'RELEASE_SEMVER'
+    displayName: 'When packing a release, define the version bump to apply'
+    type: string
+    values:
+      - major
+      - minor
+      - patch
+    default: patch
+
+# Only manual activations are intended
+trigger: none
+pr: none
+
```

```suggestion
variables:
  DEFAULT_BRANCH: $(DEFAULT_BRANCH)
  HEALTHCHECK_CONTAINER_SUBNET: $(HEALTHCHECK_CONTAINER_SUBNET)
  ${{ if eq(parameters['environment'], 'uat') }}:
    AZURE_SUBSCRIPTION: $(UAT_AZURE_SUBSCRIPTION)
    WEB_APP_NAME: $(UAT_WEB_APP_NAME)
    WEB_APP_RESOURCE_GROUP: $(UAT_WEB_APP_RESOURCE_GROUP)
    HEALTHCHECK_ENDPOINT: $(UAT_HEALTHCHECK_ENDPOINT)
    HEALTHCHECK_CONTAINER_RESOURCE_GROUP: $(UAT_HEALTHCHECK_CONTAINER_RESOURCE_GROUP)
    HEALTHCHECK_CONTAINER_VNET: $(UAT_HEALTHCHECK_CONTAINER_VNET)
  ${{ if eq(parameters['environment'], 'prod') }}:
    AZURE_SUBSCRIPTION: $(PROD_AZURE_SUBSCRIPTION)
    WEB_APP_NAME: $(PROD_WEB_APP_NAME)
    WEB_APP_RESOURCE_GROUP: $(PROD_WEB_APP_RESOURCE_GROUP)
    HEALTHCHECK_ENDPOINT: $(PROD_HEALTHCHECK_ENDPOINT)
    HEALTHCHECK_CONTAINER_RESOURCE_GROUP: $(PROD_HEALTHCHECK_CONTAINER_RESOURCE_GROUP)
    HEALTHCHECK_CONTAINER_VNET: $(PROD_HEALTHCHECK_CONTAINER_VNET)
```

add your variables as https://github.com/pagopa/hub-spid-login-ms/pull/4/files